### DISCUSSION TITLE
Placeholder duplication in pptx

### DISCUSSION THREAD

**Discussion by bpffjl** [davidgohel/officedown#98] [Q&A] [answered]
I'm trying to use a template to render a ppt, but am getting the error below.

`Error in fortify_location.location_label(location, doc = x) : 
  Placeholder "Content Placeholder 2" in the slide layout is duplicated. It needs to be unique.
Calls: <Anonymous> ... fortify_location -> fortify_location.location_label`

Looks like there is some issue with the name of a placeholder in the template. Does anyone have a good idea how to find the issue in the template? I went into the template to try to find the names of place holders, but didn't find any. Thoughts? Ideas?

**Comment by davidgohel** [comment, marked as answer]
You can find them by clicking the button where I draw a circle around in the following image:

![slide-layout-id](https://user-images.githubusercontent.com/4331618/211653441-50b4c029-6030-4640-8d28-2d6fdf9ab9f0.png)

They are displayed on the right panel.

An R solution is to look into the data returned by function `layout_properties()`:

```r
x <- read_pptx()
layout_properties(x)
```

**Reply by bpffjl** [reply]
Thanks so much!
